src/pages/Todo.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from "react";
+import Container from "@material-ui/core/Container";
+import { makeStyles } from "@material-ui/core/styles";
+import TodoForm from "../components/TodoForm";
+import TodoList from "../components/TodoList";
+import { ITodo } from "../interfaces";
+
+
+const useStyles = makeStyles({
+  root: {
+    paddingTop: "100px",
+  },
+});
+
+const Todo: React.FC = () => {
+  const classes = useStyles();
+  const [todos, setTodos] = useState<ITodo[]>([]);
+
+  useEffect(() => {
+    const saved = JSON.parse(localStorage.getItem('todos')!) as ITodo[];
```

Comment:
Could you please move  work  with localStorage in separate service. Also don't forgot about validation data